avocado/utils/configure_network.py

```diff
@@ -0,0 +1,54 @@
+"""
+Configure network when interface name and
+interface IP is available.
+"""
+import shutil
+
+from . import distro
+from . import process
+
+
+def set_ip(ipaddr, netmask, interface):
+    """
+    Gets interface name, IP, subnetmask and creates interface
+    file based on distro.
+    """
+    if distro.detect().name == 'rhel':
+        conf_file = "/etc/sysconfig/network-scripts/ifcfg-" + interface
```

Comment:
can this be like 
conf_file = "/etc/sysconfig/network-scripts/ifcfg-%s" % interface

Reply:
@PraveenPenguin 
I had initially used network.py to include this utility as well in 
https://github.com/avocado-framework/avocado/pull/3037

But this lead to checks failing in some other part of the code which I am not a owner of.

Reply:
@vaishnavibhat ah older error i am not able to locate as i think you forced pushed the PR with new change , can you please help me what error you are facing we can try to resolve